src/uproot_browser/plot_view.py

```diff
@@ -50,18 +51,16 @@ def __rich_console__(
             canvas = make_plot(self.item, width, height)
             yield rich.console.Group(*self.decoder.decode(canvas))
         except Exception:
-            rich_canvas = rich.traceback.Traceback(
+            tb = rich.traceback.Traceback(
                 extra_lines=1,
                 max_frames=4,  # Can't be less than 4 frames
             )
+            tb.max_frames = 2
+            yield tb
 
-            if options.height is not None:
-                options.height -= 4
-            yield from rich_canvas.__rich_console__(console, options)
```

Comment:
Using newly fixed textual. Limiting to two frames since it will go off screen otherwise for small screens.